test/CMakeLists.txt

```diff
@@ -61,11 +61,16 @@ list( APPEND test_output
   testoutput/tropomi_no2.nc
 )
 
+list( APPEND data
+#    Data/gdas.t12z.1bmhs.tm00
+)
+
 # create test directories and make links to the input files
 file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/testinput)
 file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/testoutput)
 file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/testrun)
-foreach(FILENAME ${test_input} ${test_output})
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/Data)
```

Comment:
remove this Data directory and put the input BUFR file (once we have a small one) in the testinput directory.

Reply:
Agreed.